components/EventForm/EventForm.js

```diff
@@ -1,9 +1,11 @@
 // Functionall imports
 
-import { useEffect } from "react";
+import { useEffect, useState } from "react";
 import { useData } from "@/lib/useData";
 import { useEventForm } from "@/lib/useEventForm";
-
+import useDebounce from "@/lib/useDebouce";
```

Comment:
Here is a little typo in your component file.

```suggestion
import useDebounce from "@/lib/useDebounce";
```